src/snowflake/cli/plugins/object/stage/commands.py

```diff
@@ -121,3 +116,48 @@ def stage_diff(
     """
     diff: DiffResult = stage_diff(Path(folder_name), stage_name)
     return ObjectResult(str(diff))
+
+
+def _get(recursive: bool, source_path: str, destination_path: str, parallel: int):
+    target = Path(destination_path).resolve()
+    if recursive:
```

Comment:
I would reverse the order `if not recursive` seems to be a shorter block. WDYT?

Reply:
For me block are the same :) , changed